pkgs/applications/graphics/mandelbulber2/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, mkDerivation, fetchFromGitHub, libpng, gsl, libsndfile, lzo,
+qmake, qttools, qtbase, qtmultimedia,
+opencl ? true, opencl-clhpp ? null, ocl-icd ? null }:
```

Comment:
`opencl` looks too much as a dependency, we usually name flags `enableFoo` or `withFoo` to distinguish them.

Reply:
Or `openclSupport`.

Reply:
Done, I went for "withOpenCL" because I felt a mild preference.